packages/qwik/src/optimizer/src/plugins/image-size-server.ts

```diff
@@ -232,11 +235,19 @@ export const getImageSizeServer = (
         }
         text = text.slice(0, offset) + imgTag + text.slice(end);
         fs.writeFileSync(filePath, text);
-      } catch (e) {
-        console.error('Error auto fixing image', e, url);
+      } else {
+        next();
+      }
+    } catch (e) {
+      if (e instanceof Error) {
+        server.ssrFixStacktrace(e);
+        await formatError(sys, e);
+      }
+      next(e);
+    } finally {
+      if (typeof (res as QwikViteDevResponse)._qwikRenderResolve === 'function') {
```

Comment:
I'm surprised this is needed, and also, shouldn't this only happen when it does something?

In any case I don't like that it's here, because this module is completely unrelated to SSR, so having this here is quite unmaintainable.